examples/cra-kitchen-sink/README.md

```diff
@@ -1 +1,42 @@
 
+# Testing
```

Comment:
If it's for contributors, it should be a part of `CONTRIBUTING.md`. Note that root jest project includes example apps, so there's no need to run `yarn test` inside package.

Ideally, image testing should be one of the available options in [`scripts/test.js`](https://github.com/storybooks/storybook/blob/release/3.3/scripts/test.js)

Reply:
I believe these are tests of the example app. These should work out of the box when using the example app. Docs are not for contributors but for users willing to start a fresh project using our CRA example.

Reply:
Maybe it's bad naming, but apps inside `example` directory are mostly for development purposes. They contain a lot of things regular app might not need

Reply:
Oh I see, but still I think this is an explanation of how to integrate storyshots with image snapshots to CRA rather than helping contributors use the feature, don't you think ?

Reply:
> this is an explanation of how to integrate storyshots with image snapshots to CRA

Then it should live in addon's documentation

Reply:
The doc inside the addon's README is already there. This is just some extra doc if someone checks the readme of this example. 
I think its harmless to leave it there.

Reply:
I think it's a bit misleading. It just describes the usage of a specific addon. If CRA setup requires some tweaks for this feature to work, this should be mentioned in addon docs

Reply:
Allright I get it, doing it now :+1: